src/components/FilterListItem/FilterListItem.jsx

```diff
@@ -21,15 +21,15 @@ class FilterListItem extends React.Component {
               className="button is-link"
               title="Edit filter"
             >
-              <span className="octicon octicon-pencil"></span>
+              ✏️
             </button>
             <button
               onClick={() => this.props.delete(this.props.filter)}
               type="button"
               className="button is-link"
               title="Delete filter"
             >
-              <span className="octicon octicon-trashcan"></span>
+              ❌
```

Comment:
It is, but I prefer an 'X' for delete rather than ignore...which raises a bigger question to me...what is the difference between snooze and ignore?